pmm-app/tests/QAN/filters_test.js

```diff
@@ -269,7 +269,7 @@ Scenario(
     I.waitForElement(qanPage.fields.noQueries, 20);
   },
 );
-
+*/
```

Comment:
@IvanaBrecherova  I am not sure why comment?

Reply:
Forgot to delete, thanks!